### PR TITLE
- metapack added, small quantapack fixes

### PR DESCRIPTION
**Pull request by AlexMdle**

### CODE REVIEW

File: src/main/resources/anniv5Resources/localization/eng/metapack/Cardstrings.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "DESCRIPTION": "If your HP is above 50%, lose !M! HP. NL Gain !M! Strength and *Vigor."
+  },
+  "${ModID}:KillerInstinct": {
+    "NAME": "Killer Instinct",
+    "DESCRIPTION": "Gain 2 Strength and 1 Dexterity. NL Startup: Shuffle a copy of this card into your draw pile.",
+    "UPGRADE_DESCRIPTION": "Gain 2 Strength and 1 Dexterity. NL Startup: Shuffle !M! copies of this card into your draw pile."
+  },
+  "${ModID}:Bulwark": {
+    "NAME": "Bulwark",
+    "DESCRIPTION": "Innate. NL Gain Temporary HP equal to your draw pile size. NL Exhaust.",
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/metapack/Cardstrings.json:39]:
```suggestion
    "DESCRIPTION": "Innate. NL Gain Temporary HP equal to the amount of cards in your draw pile. NL Exhaust.",
```

File: src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "${ModID}:RayOfFrostPower": {
+    "NAME": "Magical Protection",
+    "DESCRIPTIONS": [
+      "Channel #b",
+      " Frost whenever you play a Magic card this turn."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json:6]:
```suggestion
      " #yFrost whenever you play a #yMagic card this turn."
```

File: src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    ]
+  },
+  "${ModID}:DiehardPower": {
+    "NAME": "Diehard",
+    "DESCRIPTIONS": [
+      "Next turn, gain #b",
+      " Weak and Frail."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json:13]:
```suggestion
      " #yWeak and #yFrail."
```

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/resources/anniv5Resources/localization/eng/metapack/Cardstrings.json (hunk 1 above)
Comment (AlexMdle) [reply src/main/resources/anniv5Resources/localization/eng/metapack/Cardstrings.json:39]: This bumps the description to 6 lines. Wording is the same as Magic Dart, it should be instantly comprehensible.

File: src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json (hunk 2 above)
Comment (AlexMdle) [reply src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json:6]: Are you sure? They are already highlighted.
image

File: src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json (hunk 3 above)
Comment (AlexMdle) [reply src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json:13]: Same as ray of frost.

File: src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json (hunk 2 above)
Comment (erasels) [reply src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json:6]: This is for the power

**Review by erasels — APPROVED** [review]

**Pull request merged**